aws-lc-rs/src/rsa.rs

```diff
@@ -48,7 +50,7 @@ pub struct RsaKeyPair {
     // other thread is concurrently calling a mutating function. Unless otherwise
     // documented, functions which take a |const| pointer are non-mutating and
     // functions which take a non-|const| pointer are mutating.
-    rsa_key: LcPtr<*mut RSA>,
+    rsa_key: LcPtr<*mut EVP_PKEY>,
```

Comment:
* We should probably update the comment above. 
* Very minor nit, but should we update the variable name as well?